Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
@eirinikos Hi! You need to do `git push --force eirinikos update-srmio` in this case as you're pushing not to Homebrew's `origin` repository (as you're not (yet :wink:) a maintainer) but to your own fork. Let me know if you have any more problems.